spec/graphql/graphql-hive/analyzer_spec.rb

```diff
@@ -349,4 +349,35 @@
       )
     end
   end
+
+  context "with an invalid field" do
+    let(:query_string) do
+      %|
+      query getGatewayProjects {
+        projectsByManyTypes(type: [STITCHING]),
+        nonExistentField {
+          subField
+        }
+      }
+      |
```

Comment:
Not a blocker. This heredoc with the `~` one will ignore the `      ` in front but keep the rest of the indenting. A bit nicer if you ever need to print the query_string to the terminal.

```suggestion
      <<~GQL
      query getGatewayProjects {
        projectsByManyTypes(type: [STITCHING]),
        nonExistentField {
          subField
        }
      }
      GQL
```